proposals/exp-show-target.md

```diff
@@ -0,0 +1,210 @@
+- Enhancement Proposal PR: (leave this empty)
+- Contributors: dberenbaum, (add your Github handle)
+
+# Summary
+
+The `dvc exp show` command shows a comparison of the results of different
+experiments along with the parameters and metrics values for each experiment.
+The output currently includes all values that are defined in `params.yaml` or
+any other referenced parameters files in any DVC stage. Output also includes all
```

Comment:
> or any other referenced parameters files in any DVC stage

I think only the specific params referenced in the stage are included? (not all the params in the referened params file)
It also includes values from `vars` in dvc.yaml I think.

Reply:
> only the specific params referenced in the stage are included

Hmmm I guess not, per iterative/dvc/issues/5451.

Reply:
> It also includes values from `vars` in dvc.yaml I think.

Do you have an example of what you mean? Even if `vars` is used for substitution, wouldn't the substituted values still need to be parameters defined in `params.yaml` or another params file?

Reply:
Ah yeah never mind, I was confused 👍